Content.Server/Shuttles/Components/ThrusterComponent.cs

```diff
@@ -67,28 +68,28 @@ public sealed partial class ThrusterComponent : Component
         public TimeSpan NextFire = TimeSpan.Zero;
 
         // Frontier: upgradeable parts, togglable thrust
-        [DataField("machinePartThrust", customTypeSerializer: typeof(PrototypeIdSerializer<MachinePartPrototype>))]
-        public string MachinePartThrust = "Capacitor";
+        [DataField]
+        public ProtoId<MachinePartPrototype> MachinePartThrust = "Capacitor";
 
-        [DataField("partRatingThrustMultiplier")]
-        public float PartRatingThrustMultiplier = 1.15f; // Frontier - PR #1292 1.5f<1.15f
+        [DataField]
+        public float[] ThrustPerPartLevel = [100, 160, 210, 250];
```

Comment:
```suggestion
        public float[] ThrustPerPartLevel = [130, 170, 210, 250];
```

I think we should just up the base also, with all the changes to how far stuff are

Reply:
Changes applied.